### PR TITLE
updated input component to have some basic styling

### PR DESCRIPTION
**Pull request by ianleeduckworth**

### CODE REVIEW

File: src/design-system/atoms/Input.tsx [hunk 1]
```diff
@@ -2,13 +2,19 @@ import { InputHTMLAttributes } from "react";
 import styled from "styled-components";
 
 const StyledInput = styled.input`
-   color:orange;
-`;
+  color: white;
+  border: 2px solid gray;
+  background-color: #d7b6d7;
```
Comment (hopeduckworth) [inline src/design-system/atoms/Input.tsx]: @ianleeduckworth this purple color should be white.

**Review by hopeduckworth — CHANGES_REQUESTED** [review]

File: src/design-system/atoms/Input.tsx (hunk 1 above)
Comment (ianleeduckworth) [reply src/design-system/atoms/Input.tsx]: @hopeduckworth That makes sense. However, I'm noticing that the text color is also white which is (obviously) an accessibility issue. Should the text color maybe be changed? I can make it whatever you want, just let me know! 🚀

Comment (hopeduckworth) [reply src/design-system/atoms/Input.tsx]: Yes please!  Let's use black for now.  We may change it later but we'll keep it black for now.

Comment (ianleeduckworth) [reply src/design-system/atoms/Input.tsx]: Ok great! I've updated the colors and they have good contrast now. Thanks.

**Review by hopeduckworth — APPROVED** [review]

**Pull request merged**